app/controllers/api/pages_controller.rb

```diff
@@ -51,4 +94,11 @@ def shallow_errors(errors)
   def get_page
     @page = Page.find(params[:id])
   end
+
+  def set_language
+    @language ||= Language.find_by(code: params[:language])
+    if !params[:language].blank? && @language.blank?
```

Comment:
That doesn't work because the endpoint can be used without the language parameter, in which case pages in all languages are returned.